spec/factories/default.rb

```diff
@@ -1,8 +1,8 @@
-require 'faker'
+require "faker"
 
 FactoryBot.define do
   factory :user do
-    sequence(:name) { |n| "Josiah Carberry{n}" }
+    sequence(:name) { |_n| "Josiah Carberry{n}" }
```

Comment:
I dont think this will work now, as it's _n not n, so it'll just end up printing probably 0 or whatever defaults